banditml_pkg/banditml/serving/predictor.py

```diff
@@ -102,5 +108,119 @@ def preprocessed_input_to_pytorch(self, data):
     def predict(self, input):
         input = self.preprocess_input(input)
         pytorch_input = self.preprocessed_input_to_pytorch(input)
-        scores = self.net.predict(pytorch_input)
+        scores = self.net.forward(**pytorch_input)
```

Comment:
What's `forward` vs `predict` do? A single forward pass thru net? Does it have something to do with being able to make the scores more portable?

Reply:
exactly - forward is a forward pass through the net. we know use the pytorch net directly instead of the wrapped skorch net.